src/components/date/date.spec.tsx

```diff
@@ -515,20 +524,85 @@ describe("Date", () => {
       });
     });
 
-    describe('and with the "Tab" key', () => {
-      it('then the "DatePicker" should be closed', () => {
-        expect(wrapper.update().find(DayPicker).exists()).toBe(true);
-        simulateOnKeyDown(wrapper, "Tab");
-        expect(wrapper.update().find(DayPicker).exists()).toBe(false);
+    it('the "DatePicker" should remain open and the previous month navigation button should be focused when the "Tab" key is pressed', () => {
```

Comment:
nitpick(non-blocking): if not in a describe block then probably should just use `test` rather than `it`

This would also read better than `it` which is more of a user story structure

Reply:
I agree with the point you're making, I'd prefer to leave it and make the changes across the whole file rather than this one block as I think there's likely a fair few similar one's in the rest of the file

Reply:
I guess the reason I mentioned these ones was because it looked like new tests and only ones out of the describe block but happy to defer